tests/molecular/topology_graphs/vertex/cases/cage/conftest.py

```diff
@@ -82,9 +82,9 @@ def cls(request):
 
 @pytest.fixture(
     params=(
-        vertices._LinearCageVertex.init_at_center,
-        vertices._NonLinearCageVertex.init_at_center,
-        vertices._UnaligningVertex.init_at_center,
+        vertices.LinearVertex.init_at_center,
```

Comment:
now that the vertices are public, I think they should be accessed in tests by the public API, not by this "unofficial" API. so `stk.cage.LinearVertex` not `vertices.LinearVertex`.

Reply:
I agree and have replaced all `vertices = stk.cage.vertices` (for example) with direct calls to `stk.cage.VERTEXNAME`.